nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestSetPropertiesAction.java

```diff
@@ -80,9 +86,11 @@ public void testSetProperties() throws Exception {
         BulkStatus status = service.getStatus(commandId);
         assertNotNull(status);
         assertEquals(COMPLETED, status.getState());
-        assertEquals(10, status.getProcessed());
+        assertEquals(SIZE * 2, status.getProcessed());
```

Comment:
You should use one of other static variables because it's hard to understand why we want `SIZE * 2` if we're not already aware of the code.